### ISSUE TITLE
Suggest commands /reload and /restart cannot run in command block

### ISSUE THREAD

**Issue by HowardZHY** [CobbleSword/BurritoSpigot#20]
**Is your feature request related to a problem? Please describe.**
This could cause security problems. Wurst and other hacked clients could get written cmd-blocks with /rl and /restart and activate with redstone.

**Describe the solution you'd like**
/reload and /restart cannot run in cmd-block anymore.

**Describe alternatives you've considered**
null

**Additional context**
null

**Comment by CyberFlameGO** [comment]
This feature request can most likely be implemented via the API, so in my opinion it’s irrelevant

**Comment by sadcenter** [comment]
> This feature request can most likely be implemented via the API, so in my opinion it’s irrelevant

Exactly, `ServerCommandEvent` is a cancelable event (`BlockCommandSender` is the implementation of command block sender I think)

**Comment by HowardZHY** [comment]
I reported this to paper issues and they dont even think this is a problem

**Comment by CyberFlameGO** [comment]
Exactly, it isn't

**Comment by HowardZHY** [comment]
cmd-block's permit level is 2 , isnt it
/restart permit level should be 4 , like /stop

**Comment by CyberFlameGO** [comment]
Oh really? I thought command blocks having a lower operator level was only on later versions. Regarding your point though, I don't think /restart and /reload care what operator level the user has, unlike with /stop. It's a valid point but I don't see anyone encountering issues regarding it because the circumstances are too unique, and even if it's encountered a plugin can mitigate it

**Comment by HowardZHY** [comment]
https://minecraft.fandom.com/wiki/Permission_level
if /restart can even run with permit level 1 OP (change in ops.json) , that MUST be an PROBLEM

**Comment by crafter23456** [comment]
https://github.com/PaperMC/Paper/issues/7567 no comment besides you can only report issues there to their latest version...
disable commandblocks in your server.properties and youre fine. dont work with operators, work with permissions.
